tooling/autodispose-error-prone-checker/src/main/java/com/uber/autodispose/error/prone/checker/AutoDisposeLeakChecker.java

```diff
@@ -49,7 +49,7 @@
 @AutoService(BugChecker.class)
```

Comment:
In the future, this commit should have been two commits. One for the parallelflowable support, and another for the naming convention cleanups